### PR TITLE
DIT-967 - add new Import/Export field to BTIApplication

### PR DESCRIPTION
**Pull request by SteveSugden**

### CODE REVIEW

File: test/unit/uk/gov/hmrc/bindingtariffclassification/crypto/LocalCryptoSpec.scala [hunk 1]
```diff
@@ -32,6 +32,15 @@ class LocalCryptoSpec extends UnitSpec with MockitoSugar {
       Mockito.when(config.mongoEncryption).thenReturn(MongoEncryption(true, Some("YjQ+NiViNGY4V2l2cSxnCg==")))
       (new LocalCrypto(config)).encrypt(PlainText("hello")).toString shouldBe "Crypted(gUfxIXsmMDAbdTgm36BmEg==)"
     }
+
```
Comment (SteveSugden) [inline test/unit/uk/gov/hmrc/bindingtariffclassification/crypto/LocalCryptoSpec.scala]: (Attempt to increase code coverage)

File: build.sbt [hunk 2]
```diff
@@ -68,6 +68,6 @@ def oneForkedJvmPerTest(tests: Seq[TestDefinition]): Seq[Group] = {
 }
 
 // Coverage configuration
-coverageMinimum := 97
+coverageMinimum := 96
```
Comment (SteveSugden) [inline build.sbt]: Ed told me to reduce this!

**Comment by SteveSugden** [comment]
Could do @pllaurado - the name & values are based on a conversation with Joe but I'm open to other suggestions (I'm sure Joe doesn't really care as it's back-end)

File: test/it/uk/gov/hmrc/bindingtariffclassification/component/CaseSpec.scala [hunk 3]
```diff
@@ -105,6 +106,7 @@ class CaseSpec extends BaseFeatureSpec {
       val responseCase = Json.parse(result.body).as[Case]
       responseCase.reference shouldBe "504400001"
       responseCase.status shouldBe CaseStatus.NEW
+      responseCase.application.asInstanceOf[BTIApplication].importOrExport shouldBe None
```
Comment (ed0906) [inline test/it/uk/gov/hmrc/bindingtariffclassification/component/CaseSpec.scala]: Might be nice to have an `asBTI` method on `Application`

**Comment by ed0906** [comment]
> What do you thing to call this field **tradingActivity** instead importOrExport ?
> Then call your Enum like **TradingActivites** { IMPORT, EXPORT, ... }

Think there's lots of things we could call it, `importOrExport` is pretty descriptive though

File: test/it/uk/gov/hmrc/bindingtariffclassification/component/CaseSpec.scala (hunk 3 above)
Comment (SteveSugden) [reply test/it/uk/gov/hmrc/bindingtariffclassification/component/CaseSpec.scala]: I was looking for (and surprised that I couldn't find) a method like that.  I'll add one

Comment (SteveSugden) [reply test/it/uk/gov/hmrc/bindingtariffclassification/component/CaseSpec.scala]: Done

**Pull request merged**